imls-windows-installer/session-counter.ini

```diff
@@ -3,7 +3,8 @@
 [api]
 host=localhost:8055
 scheme=https
-uri=/items/durations/
+pres_uri=/items/durations/
```

Comment:
/items/durations isn't the right uri, is it? i think this is a leftover from directus 🤔

Reply:
Yes, you are correct, I made the adjustment before we had removed it. I will update with the new presence endpoint.